nixos/modules/services/matrix/matrix-sliding-sync.nix

```diff
@@ -44,7 +44,7 @@ in
         };
       };
       default = { };
-      description = ''
+      description = lib.mdDoc ''
```

Comment:
Is this really needed?

https://github.com/NixOS/nixpkgs/blob/3cf9ef57685162b733d8e9ae290e5d64c02e6d4f/lib/options.nix#L349

Reply:
For release-23.05 it is. Do you want me to just target release-23.05 instead?

Reply:
Ah, didn't see that backport label.